torch/_inductor/overrides.py

```diff
@@ -473,6 +561,56 @@ def fuse_binary(gm: torch.fx.GraphModule):
     return gm
 
 
+def fuse_binary_inplace(gm: torch.fx.GraphModule):
```

Comment:
Can we share code with `fuse_binary`?

Reply:
For inplace case, we have some special checks, I simplified those codes by abstracting some common code.